### PR TITLE
Implement price api

### PR DESCRIPTION
**Pull request by jnordberg**
Adds `conveyor.get_prices` call that gives STEEMSDB, STEEMUSD and STEEMVEST pairs

### CODE REVIEW

File: src/price.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import {JsonRpcError, JsonRpcMethodContext as JCtx} from '@steemit/koa-jsonrpc'
+import {Asset, Price} from 'dsteem'
+
+import {client} from './common'
+
+const ONE_STEEM = new Asset(1, 'STEEM')
+const ORDER_DEPTH = 10 // how many orders to use when calculating the STEEM<>SBD price
+
+export async function getPrices(this: JCtx) {
```
Comment (bnchdrff) [inline src/price.ts]: can we cache responses somehow?

Comment (jnordberg) [reply src/price.ts]: I was thinking we cache this via jussi

File: src/price.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ * @author Johan Nordberg <johan@steemit.com>
+ */
+
+import {JsonRpcError, JsonRpcMethodContext as JCtx} from '@steemit/koa-jsonrpc'
+import {Asset, Price} from 'dsteem'
+
+import {client} from './common'
+
+const ONE_STEEM = new Asset(1, 'STEEM')
+const ORDER_DEPTH = 10 // how many orders to use when calculating the STEEM<>SBD price
```
Comment (roadscape) [inline src/price.ts]: If not weighing by volume, not sure we gain anything by loading more than 1 order on each side. I'd recommend either keeping it dead-simple or going further w/ volume weighing.

**Comment by roadscape** [comment]
Looks great, I left just one suggestion about orderbook but not critical.

**Comment by gl2748** [comment]
This has a sister PR on `swift-steem`
https://github.com/steemit/swift-steem/pull/15

**Comment by roadscape** [comment]
Ready to merge but there seems to be a circleci hiccup

**Pull request merged**